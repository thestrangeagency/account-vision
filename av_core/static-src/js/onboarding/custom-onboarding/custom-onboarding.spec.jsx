import React from 'react';
import { shallow } from 'enzyme';

import CustomOnboarding from './custom-onboarding';

test('it renders', () => {
  const wrapper = shallow(<CustomOnboarding nextPage="" returnYear="" />);
  expect(wrapper).toMatchSnapshot();
});
