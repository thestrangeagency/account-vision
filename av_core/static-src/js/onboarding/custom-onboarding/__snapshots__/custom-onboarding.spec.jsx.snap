// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders 1`] = `
<div
  className="custom-onboarding"
>
  <div
    className="row"
  >
    <div
      className="col mb-3"
    >
      <h4
        className="my-3"
      >
        Do you have any custom expenses you’d like to enter?
      </h4>
      <p>
        Do you have any other expenses that you would like to enter that were not listed on the previous page?
      </p>
      <CustomExpenseManager
        year=""
      />
    </div>
  </div>
  <div
    className="row"
  >
    <div
      className="col"
    >
      <p
        className="mb-3"
      >
        If you are done adding expenses or prefer to add them later, click NEXT to continue.
      </p>
    </div>
  </div>
  <div
    className="row"
  >
    <div
      className="col col-6 mr-auto"
    >
      <button
        className="tx-anchor-button"
        onClick={[Function]}
        type="button"
      >
        ← Back
      </button>
    </div>
    <div
      className="col col-6"
    >
      <button
        className="btn btn-primary w-100"
        onClick={[Function]}
        type="button"
      >
        Next
      </button>
    </div>
  </div>
</div>
`;
